src/modules/flight_mode_manager/tasks/Auto/FlightTaskAuto.cpp

```diff
@@ -633,7 +634,7 @@ State FlightTaskAuto::_getCurrentState()
 		// Current position is more than cruise speed in front of previous setpoint.
 		return_state = State::previous_infront;
 
-	} else if (Vector2f(Vector2f(_position) - _closest_pt).length() > _target_acceptance_radius) {
+	} else if ((_position - _closest_pt).length() > _target_acceptance_radius) {
```

Comment:
`longerThan` ?

Reply:
I made a follow up with more of these cases here: https://github.com/PX4/PX4-Autopilot/pull/18985